OmniKit/MessageTransport/MessageBlocks/ErrorResponse.swift

```diff
@@ -8,16 +8,16 @@
 
 import Foundation
 
+public let errorResponseCode_badNonce: UInt8 = 0x14
+
 public struct ErrorResponse : MessageBlock {
     
     public let blockType: MessageBlockType = .errorResponse
     
-    public enum ErrorReponseType: UInt8 {
```

Comment:
Looks like this should rather be an enum with associated types:
```
public enum ErrorResponseType {
    case badNonce(nonceSearchKey: UInt16)
    case fault(faultEventCode: FaultEventCode, progress: PodProgressStatus)
}

Reply:
Completely reworked to use a enum case with parameters